src/odemis/driver/smarpod.py

```diff
@@ -279,9 +279,256 @@ def __init__(self, error_code):
+    # The following error limit has been exceeded during movement 
+    SA_MC_ERROR_FOLLOWING_ERROR_LIMIT_REACHED = 0x0301
+    # Positioner referencing failed 
+    SA_MC_ERROR_REFERENCING_FAILED = 0x0320
+    # Could not load required hardware driver
+    SA_MC_ERROR_DRIVER_FAILED = 0x0500
+    # Could not find/connect to controller
+    SA_MC_ERROR_CONNECT_FAILED = 0x0501
+    # The device is not connected
+    SA_MC_ERROR_NOT_CONNECTED = 0x0502
+    # The controller doesn't provide the require features or configuration
+    SA_MC_ERROR_CONTROLLER_CONFIGURATION = 0x0503
+    # Error when communicating with controller
+    SA_MC_ERROR_COMMUNICATION_FAILED = 0x0504
+
+    # property symbols
+    SA_MC_PKEY_MAX_SPEED_LINEAR_AXES = 0x00002010
+    SA_MC_PKEY_PIVOT_POINT_MODE = 0x00001001
+    SA_MC_PKEY_IS_REFERENCED = 0x00002a00
+    SA_MC_PKEY_HOLD_TIME = 0x00002000
+    SA_MC_PKEY_MAX_SPEED_LINEAR_AXES = 0x00002010
+    SA_MC_PKEY_MAX_SPEED_ROTARY_AXES = 0x00002011
+    SA_MC_PKEY_PIEZO_MAX_CLF_LINEAR_AXES = 0x00002020
+    SA_MC_PKEY_PIEZO_MAX_CLF_ROTARY_AXES = 0x00002021
+    SA_MC_PIVOT_POINT_MODE_RELATIVE = 0
+    SA_MC_PIVOT_POINT_MODE_ABSOLUTE = 1
+
+    # events
+    SA_MC_EVENT_MOVEMENT_FINISHED = 0x0001
+
+    # handles
+    # handle value that means no object
+    SA_MC_INVALID_HANDLE = 0xffffffff
+    SA_MC_INFINITE = -1
+
+    err_code = {
+0x0000: "No error",
+0x0001: "Unspecified error",
+0x0002: "Invalid parameter in function call ",
+0x0003: "Invalid locator in call to Open function ",
+0x0004: "Undefined or inaccessible property in function call ",
+0x0005: "Invalid handle in call to function ",
+0x0006: "Tried to use an unspported feature",
+0x0007: "Reached limit of simultaneously controllable devices ",
+0x0008: "Supplied buffer too small",
+0x0100: "An operation has been canceled while waiting for a result ",
+0x0101: "An operation has timed out ",
+0x0200: "The pose specified in the Move command is invalid/unreachable ",
+0x0201: "Device has not been referenced ",
+0x0203: "An operation could not be started because the device is busy ",
+0x0300: "Positioners were blocked during movement ",
+0x0301: "The following error limit has been exceeded during movement ",
+0x0320: "Positioner referencing failed ",
+0x0500: "Could not load required hardware driver",
+0x0501: "Could not find/connect to controller",
+0x0501: "Could not find/connect to controller",
+0x0502: "The device is not connected",
+0x0503: "The controller doesn't provide the require features or configuration",
+0x0504: "Error when communicating with controller",
+        }
+
+    def __init__(self):
+        if os.name == "nt":
+            raise NotImplemented("Windows not yet supported")
+            # WinDLL.__init__(self, "libSA_MC.dll")  # TODO check it works
+            # atmcd64d.dll on 64 bits
+        else:
+            # Global so that its sub-libraries can access it
+            CDLL.__init__(self, "libsmaractmc.so.0.9.1", RTLD_GLOBAL)
```

Comment:
It should be just `libsmaractmc.so.0` (which should be a link to the latest version of the library).